doc/changelog/08-tools/11394-fix-coqdoc-annotations.rst

```diff
@@ -0,0 +1,5 @@
+- **Fixed:**
+  ``coqdoc`` with option ``-g`` (Gallina only) omits all sentences with decorations
```

Comment:
The use of present tense is ambiguous: it looks like what you describe is the new (correct) behavior. What about:

>coqdoc now supports commands with attributes